aikau/src/main/resources/alfresco/util/hashUtils.js

```diff
@@ -113,6 +113,7 @@ define(["dojo/_base/array",
        * a suppresssDecoding argument of true is provided.
        *
        * @instance
+       * @function
```

Comment:
Because they're references to functions and not actually functions, JSDoc needs this tag to know that they are indeed functions. The other utils classes have these already, where appropriate, and they were just missing on this class.